src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -851,6 +856,11 @@ public BoundLiteral Literal(string value)
             return StringLiteral(stringConst);
         }
 
+        public BoundLiteral Literal(ConstantValue value, TypeSymbol type)
+        {
+            return new CSharp.BoundLiteral(Syntax, value, type) { WasCompilerGenerated = true };
```

Comment:
No, and neither is this helper needed any longer.

---
In reply to: [70734062](https://github.com/dotnet/roslyn/pull/12478#discussion_r70734062) [](ancestors = 70734062)